DESCRIPTION

```diff
@@ -1,6 +1,6 @@
 Package: austraits
 Title: Helpful functions to access the AusTraits database and wrangle data from other traits.build databases
-Version: 3.0.1
+Version: 3.0.1.9000
```

Comment:
Do we want to merge into master with a dev version? I would have thought not.

Reply:
Nice catch. Gets me everytime.